.eleventy.js

```diff
@@ -156,7 +161,8 @@ module.exports = function(config) {
   config.addFilter('githubLink', githubLink);
   config.addFilter('htmlDateString', htmlDateString);
   config.addFilter('md', md);
-  config.addFilter('paginate', paginate);
+  config.addFilter('pageContent', pageContent);
```

Comment:
Kind of a meta note. I think the intent of `addFilter` is to add stuff that you would use with the pipe operator in a nunjucks template. If these things are never used in nunjucks we might want to move them to the `_utils` folder.